packages/flutter/test/cupertino/scaffold_test.dart

```diff
@@ -26,6 +26,7 @@ void main() {
   });
 
   testWidgets('Contents padding from viewInsets', (WidgetTester tester) async {
+    BuildContext childContext;
```

Comment:
Don't augment the existing tests (do fix if they're broken). 
Instead, add a new test that specifically covers this scenario.